src/transformStateWithClones.js

```diff
@@ -8,6 +8,42 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const stateCopy = { ...state };
+  const PROPERTY_ADD = 'addProperties';
+  const PROPERTY_REMOVE = 'removeProperties';
+  const PROPERTY_CLEAR = 'clear';
+  const RESULT_OF_ACTIONS = [];
```

Comment:
seems like it is not mandatory to use UPPER_CASE while naming variables such as RESULT_OF_ACTIONS 

rather use another name, like 'actionsLog' or 'arrayOfActions' instead